src/Tribe/Admin/Notices.php

```diff
@@ -329,4 +331,60 @@ function maybe_display_paystack_notice() {
 		);
 
 	}
+
+	/**
+	 * Display AR Page notice for FSE theme detected.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function maybe_display_fse_ar_page_notice() {
+		// Bail if we aren't in Tickets > Settings.
+		if ( Settings::$settings_page_id !== tec_get_request_var( 'page' ) ) {
```

Comment:
maybe we could utilize `tribe( Settings::class )->is_on_tab( 'tab' )` here instead ?

keep in mind that tec_get_request_var is not checking for frontend vs backend and if current_user_can. So the statement could become true for a frontend request as well by an unauthorized user.

Reply:
Got it!